stories/Inputs/Multiselect.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {
+    MultiSelect,
+    MultiSelectItem,
+    MultiSelectProps,
+    MultiSelectItemProps,
+} from '../../src';
+import { createStoryTitle } from '../Constants';
+import { Meta, Story } from '@storybook/react';
+
+export default {
+    title: createStoryTitle('Multi Select'),
+    component: MultiSelect,
+    args: {
+        name: 'Demo',
+    },
+} as Meta;
+
+export const Basic: Story<MultiSelectItemProps> = (args) => (
+    <MultiSelect label={'label'} description={'Description'}>
```

Comment:
Why aren't these in args?

Reply:
I have fixed it, but when I pushed, went two more commits with it, that I didnt do it. Is that ok to merge with them?